storage-api/src/java/org/apache/hadoop/hive/ql/io/sarg/SearchArgument.java

```diff
@@ -162,6 +162,13 @@ public boolean isNeeded() {
    */
   public ExpressionTree getExpression();
 
+  /**
+   * Get the expression tree without the normalization to conjunctive normal
```

Comment:
Nit: could also explicitly say in **getExpression** above that it normalizes the expression